src/Components/Form/Form.js

```diff
@@ -16,6 +18,10 @@ export class Form extends Component {
   updateCurrentAnswers = (event) => {
     event.preventDefault();
     if (this.state.currentAnswers.includes(event.target.textContent)){
+      const answersPerQuestion = [...this.state.currentAnswers]
```

Comment:
Very sound logic here.  Thank you for making this so clean and easy to understand.